server.js

```diff
@@ -59,6 +59,63 @@ function startWorker(id) {
 
   const express = require('express');
   const Promise = require('bluebird');
+  const Profiler = require('./lib/tools/profiler');
+  const Uploader = require('./lib/tools/uploader');
+
+  /**
+   * Profiler setup
+   */
+
+  if (process.env.V8_PROFILER_ENABLED &&
+      process.env.AWS_ACCESS_KEY_ID &&
+      process.env.AWS_SECRET_ACCESS_KEY) {
+    logger.info('HEAP SNAPSHOT PROFILER IS ENABLED.');
+
+    const profiler = new Profiler({
+      interval: process.env.V8_PROFILER_DEFAULT_INTERVAL_MS,
```

Comment:
What about keeping the default values here in case the env variable doesn't exist?
```
process.env.V8_PROFILER_DEFAULT_INTERVAL_MS || 1800000 
process.env. V8_PROFILER_NAME_PREFIX || 'v8.heap.snapshot' 
```
Or updating the `.env.example` with values.. but my preference would be setting defaults and not requiring a config var to be set.